packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -183,6 +183,69 @@ Content-Type: application/json
 }
 ```
 
+However, if your SCIM server has one or more custom attributes that you want added to a new user's application profile, then any custom attributes that are defined in your application schema for user profiles are applied to that user’s application profile during the user creation operation. The request to the SCIM server looks like the following:
```

Comment:
this sentence is uber long and has past tense. Try to keep to 25 words each.

Reply:
Changed to :
There is another scenario if your SCIM server has custom attributes that you want to add for any new user. Any custom attributes defined in your application schema for user profiles are applied to the user's application profile when the user is created.